### PR TITLE
Updates to introductory material

### PR DESCRIPTION
**Pull request by sergioc**

### CODE REVIEW

File: tutorials/quickstart/tutorial.md [hunk 1]
```diff
@@ -19,6 +18,8 @@ In less than an hour, the demo retail store project will show you how to:
 
 If you've already experienced the benefits of messaging, check out our [NServiceBus step-by-step tutorial](/tutorials/nservicebus-step-by-step/).
 
+NOTE: It is recommended to keep the [glossary of messaging terms](/nservicebus/concepts/glossary) at hand to easily understand any new concepts introduced in the tutorial.
```
Comment (DavidBoike) [inline tutorials/quickstart/tutorial.md:21]:
```suggestion
NOTE: It is recommended to keep the [glossary of messaging terms](/nservicebus/concepts/glossary.md) at hand to easily understand any new concepts introduced in the tutorial.
```

File: get-started/index.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-## Start your journey with these handy resources
+### Start the journey with NServiceBus using these handy resources:
 
-* **[NServiceBus overview](https://particular.net/nservicebus)** - The most important benefits of NServiceBus at a glance.
 * **[NServiceBus technical introduction](/nservicebus/)** - A more technical introduction to NServiceBus as well as the Service Platform.
 * **[Quickstart tutorial](/tutorials/quickstart/)** - Get a tour through an NServiceBus solution covering all the elements of one-way messaging, publish-subscribe, and automatic recovery from exceptions.
-* **[Live coding your first NServiceBus system (video)](https://particular.net/webinars/live-coding-your-first-nservicebus-system)** - For those who prefer to lay back and watch an experienced developer build a messaging system with NServiceBus.
\ No newline at end of file
+* **[(Video) - Live coding your first NServiceBus system](https://particular.net/webinars/live-coding-your-first-nservicebus-system)** - For those who prefer to lay back and watch an experienced developer build a messaging system with NServiceBus.
```
Comment (DavidBoike) [inline get-started/index.md:14]:
```suggestion
* **[(Video) - Live coding your first NServiceBus system](https://particular.net/webinars/live-coding-your-first-nservicebus-system)** - For those who prefer to sit back and watch an experienced developer build a messaging system with NServiceBus.
```

File: platform/nservicebus.include.md [hunk 3]
```diff
@@ -76,3 +78,8 @@ When a message fails to be processed, even after a number of retry strategies ha
 Additionally, each endpoint can send [heartbeat](/monitoring/heartbeats/), [health check](/monitoring/custom-checks/), and [performance metrics](/monitoring/metrics/) through the platform for visualization in ServicePulse, making it easy to see which endpoints are offline, which are ready to scale out, and which require manual intervention.
 
 The [real-time monitoring demo](https://particular.net/real-time-monitoring) provides the ability to experience the Service Platform in action. The [Platform Sample package](/platform/platform-sample-package.md) provides the ability to demonstrate the Service Platform from within any Visual Studio solution, without the need to install anything.
+
+## See it in action
+
+* **[Quickstart tutorial](/tutorials/quickstart/)** - Get a tour through an NServiceBus solution covering all the elements of one-way messaging, publish-subscribe, and automatic recovery from exceptions.
+* **[(Video) Live coding your first NServiceBus system](https://particular.net/webinars/live-coding-your-first-nservicebus-system)** - For those who prefer to lay back and watch an experienced developer build a messaging system with NServiceBus.
```
Comment (DavidBoike) [inline platform/nservicebus.include.md:85]:
```suggestion
* **[(Video) Live coding an NServiceBus system](https://particular.net/webinars/live-coding-your-first-nservicebus-system)** - For those who prefer to sit back and watch an experienced developer build a messaging system with NServiceBus.
```

**Review by DavidBoike — APPROVED** [review]

File: tutorials/quickstart/tutorial.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "NServiceBus Quick Start"
-reviewed: 2022-06-28
-isLearningPath: true
```
Comment (helenktsai) [inline tutorials/quickstart/tutorial.md:4]: @sergioc Can you give some context for why you made this change? @tamararivera and I are reviewing the Quickstart now as part of her onboarding and we noticed that the link to "the learning path" shows up on Part 2 and Part 3 of the tutorial, but is missing from Part 1 and it seems a bit weird. We would expect either for it to show up on all the pages, none of the pages, or just Part 1, but not the current iteration.

Comment (sergioc) [reply tutorials/quickstart/tutorial.md:4]: Context: https://github.com/Particular/DeveloperEducation/blob/master/rfc-archive/2023-12-improve-nsb-marketing.md#docs-pages-miscellaneous-updates

Thanks for catching this! [Fixed](https://github.com/Particular/docs.particular.net/pull/6576).

**Pull request merged**